app/constants/ActionTypes.js

```diff
@@ -44,6 +44,10 @@ export default {
     SEARCH_RESULTS_GET_REQUEST: 'SEARCH_RESULTS_GET_REQUEST',
     SEARCH_RESULTS_GET_SUCCESS: 'SEARCH_RESULTS_GET_SUCCESS',
 
+    SORT_RESULTS_GET_ERROR: 'SORT_RESULTS_GET_ERROR',
+    SORT_RESULTS_GET_REQUEST: 'SORT_RESULTS_GET_REQUEST',
+    SORT_RESULTS_GET_SUCCESS: 'SORT_RESULTS_GET_SUCCESS',
```

Comment:
Are these still needed?

Reply:
No, missed to remove before. Removed now.